casadm/cas_lib_utils.c

```diff
@@ -474,6 +475,37 @@ int run_ioctl_interruptible(int fd, int command, void *cmd,
 	return ioctl_res;
 }
 
+/*
+ * Run ioctl in a way that displays progressbar (if flushing operation takes longer)
+ * Catch SIGINT signal.
+ * @param friendly_name name of management operation that shall
+ * be displayed in command prompt
+ */
+int run_ioctl_interruptible(int fd, int command, void *cmd,
+		char *friendly_name, int cache_id, int core_id)
+{
+	bool retry = false;
+
+	return run_ioctl_interruptible_retry_option(fd, command, cmd, friendly_name, 
+				cache_id, core_id, retry);
```

Comment:
can't just false be passed here instead of creating variable and passing it here?

Reply:
Ok, I've done it just for readability.